src/openfermion/transforms/_conversion.py

```diff
@@ -40,6 +41,21 @@
                                qubit_operator_sparse)
 
 
+def _check_no_sympy(operator):
```

Comment:
I'm concerned that it's too wasteful to iterate through the terms twice like this. Might be better to catch the errors inline.